### PR TITLE
feat: Sound not cover last play

### PR DESCRIPTION
**Pull request by glink25**
### Please check if the PR fulfills these requirements

- [x] The commit message follows our [guidelines](https://github.com/eva-engine/eva.js/blob/main/.github/COMMIT_MESSAGE_CONVENTION.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### Other information:

### CODE REVIEW

File: packages/plugin-sound/lib/Sound.ts [hunk 1]
```diff
@@ -87,11 +87,11 @@ class Sound extends Component {
     }
   }
 
-  play() {
+  play(uncover?: boolean) {
```
Comment (fanmingfei) [inline packages/plugin-sound/lib/Sound.ts:90]: uncover 是 揭开的意思。这里最好用一个贴切的单词。
如果不确定以后还有什么属性被添加进来，可以当作对象的属性传入

File: packages/plugin-sound/lib/Sound.ts [hunk 2]
```diff
@@ -87,11 +87,11 @@ class Sound extends Component {
...
     if (this.state !== 'loaded') {
       this.actionQueue.push(this.play.bind(this));
     }
-    this.destroySource();
+    if (!uncover)this.destroySource(); // not cover last play
```
Comment (fanmingfei) [inline packages/plugin-sound/lib/Sound.ts:94]: 这里注意一下代码lint。

**Review by fanmingfei — COMMENTED** [review]
感谢PR。

File: packages/plugin-sound/lib/Sound.ts (hunk 2 above)
Comment (glink25) [reply packages/plugin-sound/lib/Sound.ts]: 已更正

File: packages/plugin-sound/lib/Sound.ts (hunk 1 above)
Comment (glink25) [reply packages/plugin-sound/lib/Sound.ts]: 已改用overlap并设置默认初始值

**Pull request closed without merging**